src/Microsoft.ML.Data/DataLoadSave/PartitionedPathParser.cs

```diff
@@ -193,12 +196,13 @@ public IEnumerable<string> ParseValues(string path)
 
     [TlcModule.Component(Name = ParquetPartitionedPathParser.LoadName, FriendlyName = ParquetPartitionedPathParser.UserName,
         Desc = ParquetPartitionedPathParser.Summary, Alias = ParquetPartitionedPathParser.ShortName)]
-    public class ParquetPartitionedPathParserFactory : IPartitionedPathParserFactory
+    internal class ParquetPartitionedPathParserFactory : IPartitionedPathParserFactory
```

Comment:
>ParquetPartitionedPathParserFactory [](start = 19, length = 35)

Huh... weird. Not your code I know, just curious, is it obvious why this isn't living off in Parquet land?

Reply:
I don't know, and from your tone it looks like you want me to move them to Parquet project. Your wish is my command master.

---
In reply to: [252374825](https://github.com/dotnet/machinelearning/pull/2309#discussion_r252374825) [](ancestors = 252374825)

Reply:
Well, that would certainly be very nice, thank you @Ivanidzo4ka. 😄